app/utils.py

```diff
@@ -154,6 +154,34 @@ def set_url_param(url, param, value):
     return urlparse.urlunparse(url_parts)
 
 
+def set_amp_url_param(url, param, value):
```

Comment:
Is this supposed to work for URLs other than /start? I was confused because it looks it always generates URL with "amp_start". If you only intend to support /start URL:

* The function should be in start.py instead because it looks only used there.
* You can probably simply call ```self.get_url('/amp_start')``` instead.

Reply:
moved.